components/MenuBar/MenuBar.jsx

```diff
@@ -0,0 +1,76 @@
+import { PropTypes, Component } from 'react'
+
+require('./MenuBar.scss')
+
+class MenuBar extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {mobile: false}
+  }
```

Comment:
If we add `this.handleResize = this.handleResize.bind(this)` in the constructor, we can remove the code in `componentWillMount()` and just leave `window.addEventListener('resize', this.handleResize)`. I believe we don't need to put the `handleResize` method on `this.state`